src/main/java/bio/terra/service/snapshot/flight/delete/DeleteSnapshotMetadataStep.java

```diff
@@ -24,18 +23,16 @@ public DeleteSnapshotMetadataStep(SnapshotDao snapshotDao, UUID snapshotId) {
 
   @Override
   public StepResult doStep(FlightContext context) {
-    Snapshot snapshot = null;
-    boolean found = false;
+    DeleteResponseModel.ObjectStateEnum stateEnum;
     try {
-      found = snapshotDao.delete(snapshotId);
+      stateEnum =
+          snapshotDao.delete(snapshotId)
+              ? DeleteResponseModel.ObjectStateEnum.DELETED
+              : DeleteResponseModel.ObjectStateEnum.NOT_FOUND;
     } catch (SnapshotNotFoundException ex) {
-      found = false;
+      stateEnum = DeleteResponseModel.ObjectStateEnum.NOT_FOUND;
     }
```

Comment:
A minor improvement (?)
```
    DeleteResponseModel.ObjectStateEnum state = DeleteResponseModel.ObjectStateEnum.NOT_FOUND;
     try {
       if (snapshotDao.delete(snapshotId) {
         state = DeleteResponseModel.ObjectStateEnum.DELETED;
     } catch (SnapshotNotFoundException ex) {
       // It's not an error if the snapshot doesn't exist.
     }
```

Reply:
I think this reduces readability. I'm going to leave as is.